app/assets/javascripts/fileupload.js.erb

```diff
@@ -15,40 +15,14 @@
 /*global $, window, document */
 
 $(function () {
-
     'use strict';
     // Initialize the jQuery File Upload widget:
     var options = {
-      // done: function (e, data) {
-      //   alert('hola mundo :D');
-      //   ids.push(data.result.slide_id);
-      //   $('#collections_ids').val(ids);
-      // }
     };
-    <% if Rails.env.development? %>
-        // use sequential uploads in development to avoid database locking problems with sqlite3.
-      options.sequentialUploads = true;
-      options.limitConcurrentUploads = 1;
-    <% end %>
+    // use sequential uploads in development to avoid database locking problems with sqlite3.
```

Comment:
Also I think we should check for existence of the fileupload element before initializing:

```
if($('#fileupload').length > 0) {
  $('#fileupload').kiosksUploader(options);
}
```